libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -100,7 +100,7 @@ export const DealTicket = ({
   const { accountBalance: generalAccountBalance } = useAccountBalance(asset.id);
 
   const balance = (
-    BigInt(marginAccountBalance) + BigInt(generalAccountBalance)
+    BigInt(marginAccountBalance || '0') + BigInt(generalAccountBalance || '0')
```

Comment:
I don't know if changes of types in both hooks above were needed. They result in this optionality here...